utils/ifupdown.sh.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
The problem here is that I added a bashism: ${IF_MSTPCTL_MAXWAIT//[0-9]/}
${x//x/x} only works in bash and does not work in plain sh or other shells that emulate plain sh.
So, either the shebang needs to be changed to explicitly use bash, or we need to find another way to verify whether $IF_MSTPCTL_MAXWAIT is a valid number.